test/Directory.Build.targets

```diff
@@ -17,6 +17,7 @@
     <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --diagnostic --diagnostic-output-directory $(RepoRoot)artifacts/log/$(Configuration) --diagnostic-output-fileprefix $(ModuleName) --diagnostic-verbosity trace</TestRunnerAdditionalArguments>
     <TestRunnerAdditionalArguments Condition=" $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp' ">$(TestRunnerAdditionalArguments) --crashdump</TestRunnerAdditionalArguments>
     <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --hangdump --hangdump-timeout 15m</TestRunnerAdditionalArguments>
+    <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --report-azdo</TestRunnerAdditionalArguments>
```

Comment:
We have that already duplicated in `TestingPlatformCommandLineArguments`.

Arcade should be respecting `TestingPlatformCommandLineArguments` so we need to centralize everything around `TestingPlatformCommandLineArguments` instead of `TestRunnerAdditionalArguments`. That way we have consistency between running with Arcade (so far we still do it in official builds) and when running with `dotnet test`.